### ISSUE TITLE
Issue installing with Homebrew

### ISSUE THREAD

**Issue by kadreal** [codekitchen/dinghy#52]
I start by running the following command:

brew install https://github.com/codekitchen/dinghy/raw/latest/dinghy.rb

It install a few dependencies with no issue, but then I get this error:

==> Cloning https://github.com/codekitchen/dinghy.git
Error: Failed to download resource "dinghy"
Failure while executing: git clone --depth 1 --branch v3.0.4 https://github.com/codekitchen/dinghy.git /Library/Caches/Homebrew/dinghy--git

Which is weird because I can run that command myself and it seems to clone it. Whats going on here?

I'm running OSx 10.9 with homebrew 0.9.5. Not sure if it is relevant but I run RVM, and ran it with 1.8 and 2.2 of ruby (though I think homebrew uses the default ruby that the mac came packaged with?)

**Comment by codekitchen** [comment]
Yeah, homebrew uses the system ruby. Just to make sure, have you run `brew up` recently? It's too bad it doesn't give more of git's error output.

**Comment by kadreal** [comment]
Yeah, I've updated homebrew and still no luck. It's weird that git has no issues when I run it and only when homebrew does...

**Comment by codekitchen** [comment]
I think you're going to have to file an issue with homebrew on this one, sorry I can't be more help.

**Comment by kadreal** [comment]
Yeah, I'm thinking it's a homebrew issue at this point. Thanks

**Comment by kadreal** [comment]
I've fixed the problem and it is a problem with homebrew. The solution can be found [here](https://github.com/caskroom/homebrew-cask/issues/10041)

tl;dr - Homebrew does not have a hardlink set up correctly to system Ruby, which causes it to fail when it tries to run a ruby script to clone the repo.
